packages/playwright-core/src/server/transport.ts

```diff
@@ -88,8 +90,9 @@ export class WebSocketTransport implements ConnectionTransport {
     return transport;
   }
 
-  constructor(progress: Progress|undefined, url: string, headers?: { [key: string]: string; }, followRedirects?: boolean) {
+  constructor(progress: Progress|undefined, url: string, logUrl: string, headers?: { [key: string]: string; }, followRedirects?: boolean) {
     this.wsEndpoint = url;
+    this._logUrl = logUrl;
```

Comment:
nit: I'd compute logUrl right here:

```suggestion
    this._logUrl = stripQueryParams(url);
    progress?.log(`<ws connecting> ${logUrl}`);
```